interactions/api/gateway/client.py

```diff
@@ -507,12 +505,10 @@ def __option_type_context(self, context: object, type: int) -> dict:
     async def restart(self):
         await self.__restart()
 
-    @property
-    async def __receive_packet_stream(self) -> Optional[Dict[str, Any]]:
+    async def __get_next_packet(self) -> Union[Dict[str, Any], WSMessage, None]:
```

Comment:
It was meant to be a property, please keep it as that

Reply:
A property being asynchronous is... odd, to be fair. I can see why it was changed to be a function.

I'm not *against* keeping it as a property per se, but what was the reason for making it a property?

Reply:
uh.. @jameswalston? ..